soc/arm/st_stm32/stm32l1/Kconfig.defconfig.stm32l151x8a

```diff
@@ -0,0 +1,31 @@
+# Kconfig - ST Microelectronics STM32L151X8-A MCU
+#
+# Copyright (c) 2019 eTactica ehf
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_STM32L151X8A
```

Comment:
You need to actually define this Kconfig symbol in `soc/arm/st_stm32/stm32l1/Kconfig.soc`

Reply:
ok, done. easy to miss that thanks.